app/precompiled.py

```diff
@@ -570,6 +574,25 @@ def rewrite_address_block(pdf, *, page_count, allow_international_letters):
 
 
 def _extract_text_from_pdf(pdf, *, x1, y1, x2, y2):
```

Comment:
shouldn't it be called `_extract_text_from_first_page_of_pdf` or something?

Reply:
great point - squashed into the refactor commit